tools/packaging/release/update-repository-version.sh

```diff
@@ -244,6 +257,29 @@ EOT
 		out=""
 		out=$("${hub_bin}" pull-request -b "${target_branch}" -F "${notes_file}" 2>&1) || echo "$out" | grep "A pull request already exists"
 	fi
+
+	if [ "${repo}" == "kata-containers" ] && [ "${target_branch}" == "main" ] && [[ "${new_version}" =~ "rc" ]]; then
+		reverting_kata_deploy_changes_branch="revert-kata-deploy-changes-after-${new_version}-release"
+		git checkout -b "${reverting_kata_deploy_changes_branch}"
+
+		git revert --no-edit ${kata_deploy_commit} >>/dev/null
+		commit_msg="$(generate_revert_kata_deploy_commit $new_version)"
+		info "Creating the commit message reverting the kata-deploy changes"
+		git commit --amend -s -m "${commit_msg}"
+
+		echo "${commit_msg}" >"${notes_file}"
+		echo "" >>"${notes_file}"
+		echo "Only merge this commit after ${new_version} release is successfully tagged!" >>"${notes_file}"
+
+		if [[ ${PUSH} == "true" ]]; then
+			info "Push \"${reverting_kata_deploy_changes_branch}\" to fork"
+			${hub_bin} push fork -f "${reverting_kata_deploy_changes_branch}"
+			info "Create \"${reverting_kata_deploy_changes_branch}\" PR"
+			out=""
+			out=$("${hub_bin}" pull-request -b "${target_branch}" -F "${notes_file}" 2>&1) || echo "$out" | grep "A pull request already exists"
```

Comment:
Looking at the examples of pull requests that you put in https://github.com/kata-containers/kata-containers/pull/3089#issuecomment-979065975 ; a new idea came to my mind: when open the PR this script could already set the "no-backport/forward" labels so the labels checker doesn't fail (also maybe set a 'release' label too).

Feel free to send a follow up PR if that idea makes sense.

Reply:
@fidencio that's the only remaining comment that I have. Everything else seems good on this PR.

Reply:
@wainersm, as this is something that affects how things are **nowadays**, I'd prefer having this treated in a different PR, which I will then backport to the stable-2.3 branch.

Reply:
Fair enough :)